flink-core/src/main/java/org/apache/flink/configuration/AkkaOptions.java

```diff
@@ -39,14 +42,24 @@
                                     + "memory footprint.");
 
     /** Timeout for akka ask calls. */
-    public static final ConfigOption<String> ASK_TIMEOUT =
+    public static final ConfigOption<Duration> ASK_TIMEOUT_DURATION =
```

Comment:
Looking at the usages, I wondered why we use `Duration` in the first place instead of `Time`. I saw only two usages that actually use the Duration class. One is in `AkkaUtils` (and could be done with `Time` as well), the other in `WebMonitorEndpoint` (which I didn't follow further to see how it relies on `Duration`).
Is it because `Time` is marked `@PublicEvolving`, so potentially unstable?

Reply:
We actually want to get rid of `Time` eventually because `Duration` provides the same functionality. FLINK-14068